ctms/sample_data.py

```diff
@@ -177,7 +177,8 @@ class ContactVendor:
     }
 
     def __getitem__(self, key: str) -> ContactSchema:
-        return self.contacts[key][0].copy(deep=True)
+        contact: ContactSchema = self.contacts[key][0].copy(deep=True)
+        return contact
```

Comment:
> ctms/sample_data.py:180: error: Returning Any from function declared to return "ContactSchema"
> Found 1 error in 1 file (checked 2 source files)
Fix for this mypy pre-commit issue which came up?

Reply:
Maybe! I don't see this error in my dev environment, however, so I can't tell for sure.

The sample data needs to be moved out of ``ctms`` and into ``tests/unit``, but that's for a future PR.